apps/meteor/app/integrations/server/methods/outgoing/updateOutgoingIntegration.ts

```diff
@@ -96,7 +95,7 @@ Meteor.methods<ServerMethods>({
 					triggerWordAnywhere: integration.triggerWordAnywhere,
 					runOnEdits: integration.runOnEdits,
 					_updatedAt: new Date(),
-					_updatedBy: Users.findOne(this.userId, { fields: { username: 1 } }),
+					_updatedBy: await Users.findOne(this.userId, { projection: { username: 1 } }),
```

Comment:
I thougt findOne received an object 🤔 it's customized on the model?

Reply:
No, you are right, there's a lot of confusion with those methods because the previous version (sync) was allowing queries like this one, receiving just the id on findOne, update, etc. Will change, tks.